### PR TITLE
Added icub-main formula

### LINKED ISSUE #3: icub-main formula
We should create a formula for `icub-main` since `ipopt` and `opencv3` have been added in `brew`.

I'm currently using them and they seems to work just fine!

cc: @francesco-romano

### PR DESCRIPTION
**Pull request by francesco-romano**
Fixes #3

### CODE REVIEW

File: Formula/icub-main.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+class ICUBMain < Formula
```
[the remark this answers is not in the document]
Comment (francesco-romano) [reply Formula/icub-main.rb]: Strange. I was sure I did the test.
Changed to `IcubMain`.
I will now push

[the remark this answers is not in the document]
Comment (claudiofantacci) [reply Formula/icub-main.rb]: Maybe something changed under the ruby hood.
Not a big deal 😄

**Comment by francesco-romano** [comment]
Yes.. not a big deal... Done.
Do a last check if I did not miss something, please. Then you can directly merge.

**Comment by francesco-romano** [comment]
Thanks @claudiofantacci

**Pull request merged**